src/components/displaybooks.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import style from './displaybook.module.css';
+
+const DisplayBooks = (props) => (
+  <div className={style.book}>
+    <div className={style.aboutBook}>
+      <p>{props}</p>
+      <h2>{props}</h2>
+      <a href="#head">{props}</a>
+      <div className={style.bookAction}>
+        <a href="#comments">Comments</a>
+        {' '}
+        |
+        <a href="#remove">Remove</a>
+        |
+        <a href="#edit">Edit</a>
+      </div>
+    </div>
+
+    <div className={style.progress}>
+      <progress value={props} max="100" />
+      <div className={style.progressDetail}>
+        <h4>{props}</h4>
+        <span>Completed</span>
+      </div>
+    </div>
+
+    <div className={style.currentChapter}>
+      <p>Current Chapter</p>
+      <h4>{props}</h4>
+      <button type="button">Update Progress</button>
+    </div>
+  </div>
+);
```

Comment:
- [ ] Hello @krisbobo, I think you are missing the point here. The changes you need to make are simple and if not implemented the app won't display anything in the browser, it's supposed to run successfully and display, a navigation header, book, and a form for adding a book(no functionality yet). 

**Here is a screenshot of the app when I ran it. It shows an empty screen and errors in the console.**

<img width="1774" alt="Screenshot 2023-02-25 at 10 47 41 PM" src="https://user-images.githubusercontent.com/50941074/221383424-8c0e0876-79a0-45df-a96b-47b5f85ea8e9.png">

**Here is the edit I made to the code(If you can, please be sure you pass the right value. I just did basic edit just for things to display)**

<img width="591" alt="Screenshot 2023-02-25 at 10 54 40 PM" src="https://user-images.githubusercontent.com/50941074/221383500-cf6d7080-6348-4815-bbb7-29c176951809.png">

**Here is the output of the edited code in the browser with just a single warning in the console for default props. Ideally, this is what is expected to display 👍🏽**

<img width="1796" alt="Screenshot 2023-02-25 at 10 53 48 PM" src="https://user-images.githubusercontent.com/50941074/221383565-f5ee550b-5ac9-458a-9ab2-024a114c4b8f.png">